Tests/TestingTests/EventRecorderTests.swift

```diff
@@ -178,6 +178,44 @@ struct EventRecorderTests {
         .first != nil
     )
   }
+  
+  @available(_regexAPI, *)
+  @Test(
+    "number of arguments based on verbosity level at the end of test run",
```

Comment:
The display name of this test can be rephrased now to not mention verbosity. I'd also prefer if we could capitalize the first word to match prevailing style.